Core/src/org/sleuthkit/autopsy/guiutils/WrapLayout.java

```diff
@@ -0,0 +1,607 @@
+     * instance, if items should be displayed left to right based on locale,
+     * these components will be on the right.
+     *
+     * @return The components to display with opposite alignment.
+     */
+    public Collection<Component> getOppositeAlignedItems() {
+        return oppositeAlignedItems;
+    }
+
+    /**
+     * Gets the horizontal gap between components and between the components and
+     * the borders of the <code>Container</code>
+     *
+     * @return The horizontal gap between components and between the components
+     *         and the borders of the <code>Container</code>.
+     */
+    public int getHgap() {
+        return hgap;
+    }
+
+    /**
+     * Sets the horizontal gap between components and between the components and
+     * the borders of the <code>Container</code>.
+     *
+     * @param hgap The horizontal gap between components and between the
+     *             components and the borders of the <code>Container</code>.
+     */
+    public void setHgap(int hgap) {
+        this.hgap = hgap;
+    }
+
+    /**
+     * Gets the vertical gap between components and between the components and
+     * the borders of the <code>Container</code>.
+     *
+     * @return The vertical gap between components and between the components
+     *         and the borders of the <code>Container</code>.
+     */
+    public int getVgap() {
+        return vgap;
+    }
+
+    /**
+     * Sets the vertical gap between components and between the components and
+     * the borders of the <code>Container</code>.
+     *
+     * @param vgap The vertical gap between components and between the
+     *             components and the borders of the <code>Container</code>.
+     */
+    public void setVgap(int vgap) {
+        this.vgap = vgap;
+    }
+
+    /**
+     * Sets whether or not components should be vertically aligned along their
+     * baseline. Components that do not have a baseline will be centered. The
+     * default is false.
+     *
+     * @param alignOnBaseline Whether or not components should be vertically
+     *                        aligned on their baseline.
+     */
+    public void setAlignOnBaseline(boolean alignOnBaseline) {
+        this.alignOnBaseline = alignOnBaseline;
+    }
+
+    /**
+     * Returns true if components are to be vertically aligned along their
+     * baseline. The default is false.
+     *
+     * @return true If components are to be vertically aligned along their
+     *         baseline.
+     */
+    public boolean getAlignOnBaseline() {
+        return alignOnBaseline;
+    }
+
+    /**
+     * Adds the specified component to the layout. Not used by this class.
+     * NOTE: This is not used for this layout
+     *
+     * @param name The name of the component.
+     * @param comp The component to be added.
+     */
+    @Override
+    public void addLayoutComponent(String name, Component comp) {
+    }
```

Comment:
This doesn't feel like something we want to add to the public API a method where it does nothing and just has a note saying it is not used for this layout in the comment.

Also codacy doesn't like the empty body being undocumented.

Reply:
This is a method of the interface, java.awt.LayoutManager.  I documented that it was not in use in the method body.  Is there a particular way this should be documented?